libs/base/form/src/mixins/form.mixin.ts

```diff
@@ -19,6 +19,7 @@ export declare class FormMixinInterface<FormValues = unknown>
 export const FormMixin = <
   // TODO: remove any and pass generic type to mixin
   // when we fix issue with types inference of mixins chain
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
can we make it `unknown`?

Reply:
Unfortunately no, because `unknown` required providing some type to be able to access to the typed fields and providing some specific type for the parameter brake the mixin classes chain. We have such issue and it described above.